python/lsst/ip/isr/ptcDataset.py

```diff
@@ -693,6 +695,21 @@ def fromTable(cls, tableList):
                 inDict['ptcTurnoffSamplingError'][ampName] = np.nan
             else:
                 inDict['ptcTurnoffSamplingError'][ampName] = record['PTC_TURNOFF_SAMPLING_ERROR']
+            if calibVersion < 1.9 and inDict['ptcFitType'] == "FULLCOVARIANCE":
+                # Before version 1.9, the noise stored in the PTC was in
+                # units of electron^2 only if ptcFitType == FULLCOVARIANCE.
+                # After version 1.9, we standardized the
+                # PhotonTransferCurveDataset noise units to electron to fix
+                # this bug. If a user tries to use an earlier version of
+                # PTC with this fit type, we must be sure to do the
+                # calculations properly. More information about this noise
+                # issue can be found in DM-45976.
+                cls().log.info(f"Input PTC VERSION ({calibVersion}) < 1.9 and"
+                               " ptcFitType == FULLCOVARIANCE. Applying fix"
+                               f" for DM-45976 noise issue for AMP {ampName}.")
```

Comment:
I think that we should put extra logic in here because we don't need 16x this message, only once not per amp.  Or make this a `debug` log.  Up to you.

Reply:
I'll keep it an info log, but I'll only have it output once